palette/src/oklab.rs

```diff
@@ -125,6 +140,10 @@ impl<T> Oklab<T> {
     }
 }
 
+//FIXME:https://colorjs.io/docs/spaces.html#oklab uses different values to the documentation above and these min- and max- values
+// From my understanding Chroma in Oklab is unlimited, as it is only limited, when used in reference to the gamut of a specific  display technology.
+// So here, a and b should be unlimited. Only in Okhsl and Okhsv should chroma be limited, as they reference the sRGB gamut.
+// For a different Okhsl and Okhsv, which references HDR (https://bottosson.github.io/posts/colorpicker/#ideas-for-future-work) they'd need to be different again.
```

Comment:
A note on HDR is that Palette doesn't try to handle it in any special way at the moment. It's a plus if it happens to work but not necessary for now. Let's start with sRGB for now. A future extension could be to look into supporting other gamuts, maybe by generalizing over `RgbSpace` if that's sufficient.

Reply:
So should I just remove this fixme and be done with it or should I add a comment somewhere? And if so, what should it say and where would I put it?

Reply:
You can mention in the implementation, where it adjusts to the gamut, that we assume sRGB. Maybe also in the descriptions of `Okhsv`, `Okhsl` and `Okhwb` so it's clear what they represent. Otherwise, it's enough to refer to the reference implementation so there's something to compare to when debugging or reworking something. I appreciate that you have already added comments with clarifications in some places! It helps a lot!